src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
+@import "./utils/variables.scss";
+@import "./utils/extends.scss";
+
+@import "./blocks/header.scss";
+@import "./blocks/logo.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/container.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+* {
```

Comment:
Where are this files?

Reply:
Sorry, I don't know how it happened. Did everything as usual... 

I've already added all the necessary files